### ISSUE TITLE
People: Jetpack sites prefix email addresses with "@"

### ISSUE THREAD

**Issue by sixhours** [Automattic/wp-calypso#32479]
<img width="745" alt="Screen Shot 2019-04-22 at 2 44 43 PM" src="https://user-images.githubusercontent.com/2124984/56518455-7f865100-650d-11e9-8241-74a6ea9d1b46.png">

Noticed while working on #32397. To reproduce this:

* Go to People for a Jetpack-connected site.
* Note the administrator (in this case) has an "@" symbol before the email address, reading "@calobee@gmail.com" which looks odd.

**What I expected**

The display name should only prefix usernames with the @ symbol, not addresses or proper names.

**Comment by donpark** [comment]
@sixhours Can users of Jetpack sites use their email address as login name? Field rendered is `user.login`.

Superficial fix is to just omit `@` if login looks like an email address.

**Comment by donpark** [comment]
Just verified that I could use email address as admin login name. Even `@@@@donpark@@@@` worked.

So omitting `@` prefix could lead to referencing wrong login name.

**Comment by sixhours** [comment]
Yeah, it looks like my login for that site is my email address. I wonder, is there a good reason to include the @ in front of the username for display? I'm guessing it's intended as a symbol to visually indicate "Hey, this is a username!" but maybe prefixing with something like "Username:" would be clearer (and less likely to cause bugs in pattern matching).

I mocked it up and I hate it, visually :D but it's probably less error-prone:

<img width="761" alt="Screen Shot 2019-04-25 at 3 52 16 PM" src="https://user-images.githubusercontent.com/2124984/56764233-3d117e00-6772-11e9-9dda-6a299dabf789.png">

Alternatively, would it be too confusing to remove the @ symbol altogether?

<img width="758" alt="Screen Shot 2019-04-25 at 3 52 42 PM" src="https://user-images.githubusercontent.com/2124984/56764291-5e726a00-6772-11e9-8323-05aa0d32b447.png">

**Comment by donpark** [comment]
> Alternatively, would it be too confusing to remove the @ symbol altogether?

To me, yes. Without the @ prefix, it looks more like a company name.

**Comment by alaczek** [comment]
Agreed that the "@" sign in front of username is confusing. I'd be for your suggestion to add "Username:" instead.

**Comment by eeeeevon13** [comment]
I looked at the proposed solutions above. Neither seems ideal, as adding the label "username" adds a lot of extra noise to the interface, which might have the opposite effect that we are after for this (clarity). We also explored adding some logic that checks the user name for the "@" symbol and would just not add it if there is one already present (as in an email address)

My main issue with all of these solutions is that they deviate from the most common way user names are referred to generally in the wild (Twitter, Slack, Facebook etc.)  

I checked with our data team and it appears that the instances of users with an email address as their username is very very low. We're looking at a total of 123. I think at this point we should just not consider this a bug/bottle and readdress if it comes to light that this is actually an issue affecting a large enough audience to merit iteration.
